manifest.jps

```diff
@@ -328,7 +329,15 @@ actions:
     if (${env.status} == 1):
       cmd [${targetNodes.nodeGroup}]: sed -i 's|.*bitninja.*||g' /etc/jelastic/redeploy.conf
       user: root
-    
+
   disableIPForwarding:
-    cmd [${targetNodes.nodeGroup}]: echo 0 > /proc/sys/net/ipv4/ip_forward
+    - if (${globals.ip_forward:} == ""):
+        - cmd [${targetNodes.nodeGroup}]: echo $(cat /proc/sys/net/ipv4/ip_forward)
+        - setGlobals:
+            ip_forward: ${response.out}
```

Comment:
this part is not related to disableIPForwarding

Reply:
fixed